### PR TITLE
Updates to self-hosted

### PR DESCRIPTION
**Pull request by jamessewell**
- Add CTA to all self-hosted pages
- Update Kubernetes page to reflect Helm not being maintained

### CODE REVIEW

File: _partials/_cloudtrial.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <Highlight type="cloud" header="Try for free on Timescale" button="Try for free">
-Your Timescale trial is completely free for you to use for the first
-thirty days. This gives you enough time to complete all the tutorials and run a
-few test projects of your own.
+<ul>
+<li> get started at the click of a button </li>
```
Comment (billy-the-fish) [inline _partials/_cloudtrial.md]: Get ...

File: _partials/_cloudtrial.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+<li> get access to advanced cloud features like transparent bottomless object storage </li>
```
Comment (billy-the-fish) [inline _partials/_cloudtrial.md]: Access advanced cloud ...

File: _partials/_cloudtrial.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+<li> don't waste time running high performance, highly available TimescaleDB and PostgreSQL in the cloud </li>
```
Comment (billy-the-fish) [inline _partials/_cloudtrial.md]: I read this as don't waste time using Timescale. Can you rephrase please.

File: _partials/_selfhosted_cta.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<Highlight type="cloud" header="Want to skip these steps?" button="Try for free">
+Deploy a Timescale service in the cloud. Don't worry about performance, scalability, high availability, backups or management.
```
Comment (billy-the-fish) [inline _partials/_selfhosted_cta.md]: ... in the cloud. We tune your service for performance ... so you can sit back and relax.

File: _partials/_timescaledb.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+TimescaleDB is an extension for PostgreSQL that enables time-series workloads, 
+increasing ingest, query, storage and analytics performance. 
+We recommend running TimescaleDB on Timescale in the cloud,
```
Comment (billy-the-fish) [inline _partials/_timescaledb.md]: Best practice is to run Timescale ... .  If you want to self host, you easily add Timescale functionality to your PostgreSQL instance.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Looks good, a couple of small style suggestions.

File: _partials/_cloudtrial.md (hunk 1 above)
Comment (jamessewell) [reply _partials/_cloudtrial.md:3]: This one isn't actually used anymore :) I left it in the partials directory for reference

File: _partials/_cloudtrial.md (hunk 2 above)
Comment (jamessewell) [reply _partials/_cloudtrial.md:4]: This one isn't actually used anymore :) I left it in the partials directory for reference

File: _partials/_cloudtrial.md (hunk 1 above)
Comment (jamessewell) [reply _partials/_cloudtrial.md]: This is the old one, I've actually reverted b/c my changes don't use it and it leaks out. 

We can attack CTAs in general in another PR

File: _partials/_cloudtrial.md (hunk 2 above)
Comment (jamessewell) [reply _partials/_cloudtrial.md]: This is the old one, I've actually reverted b/c my changes don't use it and it leaks out. 

We can attack CTAs in general in another PR

File: _partials/_cloudtrial.md (hunk 3 above)
Comment (jamessewell) [reply _partials/_cloudtrial.md]: This is the old one, I've actually reverted b/c my changes don't use it and it leaks out. 

We can attack CTAs in general in another PR

**Review by billy-the-fish — APPROVED** [review]
Nice docs.
